util/util.go

```diff
@@ -15,7 +18,19 @@ func WriteConfigmap(clientset *kubernetes.Clientset, permission map[string]map[s
 	if err != nil {
 		return err
 	}
+	data := strings.Replace(string(permissions), "\n", "\\n", -1)
+	patch := []byte(fmt.Sprintf(`{"data":{"labels.yaml": "%s"}}`, data))
 
+	_, err = clientset.CoreV1().ConfigMaps(c.CMNamespace).Patch(context.Background(), c.CMName, types.MergePatchType, patch, metav1.PatchOptions{})
```

Comment:
Does the patch clear all fields that are not included in the 'to be patched' map? Otherwise, users could only gain permissions but not lose them.

Reply:
it does. obviously i tested the implementation ;-)